docs/README.md

```diff
@@ -1,114 +1,232 @@
 # ScalarDB Benchmarks
```

Comment:
Currently, the title can be read in a way that users might expect that the document describes benchmarking we've done rather than showing users how to run benchmarks.

How about we change this title as follows?

```suggestion
# ScalarDB Benchmarking Tools
```